### PR TITLE
[WIP] remove some unnecessary backgrounds

### PR DESCRIPTION
**Pull request by rasom**
attempt to fix redrawing https://discuss.status.im/t/before-i-jump-on-the-react-native-is-slow-bandwagon/1064

before/after

intro

welcome

wallet

chat_list

my_profile

chat

### CODE REVIEW

File: mobile_files/package.json.orig [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "react-native-udp": "git+https://github.com/status-im/react-native-udp.git#2.3.1-1",
     "react-native-webview-bridge": "git+https://github.com/status-im/react-native-webview-bridge.git#0.33.16-4",
     "react-navigation": "^2.12.1",
+    "react-navigation-stack": "git+https://github.com/status-im/react-navigation-stack#6d1133b25c4d6ba059cffe7ccf69c92da628a448",
```
Comment (yenda) [inline mobile_files/package.json.orig:64]: what is this for?

Comment (rasom) [reply mobile_files/package.json.orig:64]: https://github.com/status-im/react-navigation-stack/commit/6d1133b25c4d6ba059cffe7ccf69c92da628a448

Comment (yenda) [reply mobile_files/package.json.orig:64]: ok I see it is a transitive dependency of react-navigation
probably worth pushing upstream if we stick with it

Comment (yenda) [reply mobile_files/package.json.orig:64]: nice catch!

Comment (rasom) [reply mobile_files/package.json.orig:64]: we need to upgrade react-navigation in out project first

**Comment by rasom** [comment]
This PR will be split into several small PRs in order to minimize possible regressions.

**Pull request closed without merging**